cobbler/api.py

```diff
@@ -1065,26 +1048,24 @@ def get_signatures(self) -> dict:
         """
         return utils.SIGNATURE_CACHE
 
-    def signature_update(self, logger):
+    def signature_update(self):
         """
         Update all signatures from the URL specified in the settings.
-
-        :param logger: The logger to audit the removal with.
         """
         try:
             url = self.settings().signature_url
-            dlmgr = download_manager.DownloadManager(self._collection_mgr, self.logger)
+            dlmgr = download_manager.DownloadManager(self._collection_mgr)
             # write temp json file
             tmpfile = tempfile.NamedTemporaryFile()
             sigjson = dlmgr.urlread(url)
             tmpfile.write(sigjson.text.encode())
             tmpfile.flush()
-            logger.debug("Successfully got file from %s" % self.settings().signature_url)
+            self.logger.debug("Successfully got file from %s" % self.settings().signature_url)
             # test the import without caching it
             try:
                 utils.load_signatures(tmpfile.name, cache=False)
             except:
-                logger.error("Downloaded signatures failed test load (tempfile = %s)" % tmpfile.name)
+                self.logger.error("Downloaded signatures failed test load (tempfile = %s)" % tmpfile.name)
```

Comment:
```suggestion
                self.logger.error("Downloaded signatures failed test load (tempfile = %s)" tmpfile.name)
```
The `%` Could be omited according to https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation#34634301